src/hb-ot-layout-common-private.hh

```diff
@@ -479,6 +479,13 @@ struct FeatureParams
     return Null(FeatureParamsSize);
   }
 
+  inline int get_stylistic_set_name_id (hb_tag_t tag) const
```

Comment:
Does this work with `cvXX` features, they can have name IDs as well. Also I’d rename it `get_feature_name_id` to be more general.

Reply:
There are multiple name id associated with them https://docs.microsoft.com/en-us/typography/opentype/spec/features_ae#cv01-cv99 What do you suggest?

Reply:
Used featUILableNameID but probably some API is needed to get all of them.